src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -210,4 +210,7 @@ Temporarily using the Windows Terminal default settings.
   <data name="CloseWindowWarningTitle" xml:space="preserve">
     <value>Do you want to close all tabs?</value>
   </data>
-</root>
+  <data name="InvalidBackgroundImage" xml:space="preserve">
+    <value>Found a profile with an invalid "backgroundImage". Defaulting that profile to have no background image. Make sure that when setting a "backgroundImage", the value is a valid file path to an image.</value>
+  </data>
+</root>
```

Comment:
There's a good amount of "changes" here that actually don't exist. Please be sure to fix that.